transcom/mymove
# B 21468 history audit counseling office

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1015734&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## [INT PR](https://github.com/transcom/mymove/pull/13947)

## Summary

We need to track all new updates of assigning and updating a counseling office to a move and add them to the Move history tab and display properly.
Also, some tests were failing from some other files, files I previously touched in a story a few weeks ago, so went back and adjusted some files and broke the audit logs out to not use the '*' wildcard for event name.  All files have to do with creating and updating an order, so all will be touched in the steps below.

### How to test

1. Login as a milmove customer, create an order, picking a non-prime counseled duty location, take note of the Counseling Office chosen.  Save the orders info and at least go through the process to upload an Orders document (don't need to create a shipment).
2. Go back to move created in step 1, Edit the orders, and change the Counseling office.
3. Go through step 1 again but pick a prime counseled location (NS Mayport).
4. Login as an office user and search for the move from step 1 and verify that the Counseling office displays properly in the Move history tab.
5. Search for the move from step 3 and verify that Counseling office does not show up since it was not available for a prime counseled move.
